src/intervalSequence.ts

```diff
@@ -129,3 +136,31 @@ export const merge
             }
         return result
     }
+
+export type Add
+    =  <E, A>(_: Strategy<E, A>)
+    => (_: IntervalSequence<E, A>)
+    => (_: Interval<E, A | undefined>)
+    => IntervalSequence<E, A>
+
+export const add
+    : Add
+    = intervalSequence => current => toAdd => {
+        type EA = typeof current extends IntervalSequence<infer _E, infer _A>
+            ? readonly [_E, _A] : never
+        type E = EA[0]
+        type A = EA[1]
+
+        const mergeSeq: IntervalSequence<E, A | undefined> = {
+            first: undefined,
+            rest: sequence.fromArray([
+                { edge: toAdd.min, value: toAdd.value },
+                { edge: toAdd.max, value: undefined }
+            ])
+        }
+
+        return merge<E, A, A | undefined, A>({
```

Comment:
please, avoid type annotations in function calls. It's not compatible with current fun.ts language. use
```
const NAME
    : TYPE_DEFINITION
   = VALUE
return NAME
```

Reply:
fixed